suites/pacific/cephfs/tier-2_cephfs_test-nfs.yaml

```diff
@@ -0,0 +1,146 @@
+---
+#=======================================================================================================================
+# Tier-level: 2
+# Test-Suite: tier-2_cephfs_test-nfs.yaml
+# Conf file : conf/pacific/cephfs/tier_0_fs.yaml
+# Test-Case Covered:
+# CEPH-83574028 - Ensure the path of the nfs export is displayed properly.
+# CEPH-83574024 - Ensure Snapshot and cloning works on nfs exports
+# CEPH-83574027 - Ensure creation of Subvolgroups, subvolumes works on NFS exports and run IO from nfs clients
+# CEPH-83574003 - Export the nfs share with cli with RO access
+#=======================================================================================================================
+tests:
+  -
+    test:
+      abort-on-fail: true
+      desc: "Setup phase to deploy the required pre-requisites for running the tests."
+      module: install_prereq.py
+      name: "setup install pre-requisistes"
+  -
+    test:
+      abort-on-fail: true
+      config:
+        steps:
+          -
+            config:
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                registry-url: registry.redhat.io
```

Comment:
Please avoid `registry-url`. This value is different when the suite is executed in the pipeline